scripts/assets/test_file_exemptions.textproto

```diff
@@ -371,6 +371,7 @@ exempted_file_path: "app/src/main/java/org/oppia/android/app/recyclerview/StartS
 exempted_file_path: "app/src/main/java/org/oppia/android/app/resumelesson/ResumeLessonActivityPresenter.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/resumelesson/ResumeLessonFragmentPresenter.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/resumelesson/ResumeLessonViewModel.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/settings/profile/DeleteProfileSuccessDialogFragment.kt"
```

Comment:
We actually should have a test for the fragment, too, and this may influence how you test ProfileEditFragment.

ProfileEditFragment's tests technically only care about whether the dialog shows up when expected, and that the cancel/delete buttons do what's expected (i.e. closes the dialog).

DeleteProfileSuccessDialogFragment's tests, on the other hand, should verify properties regarding the fragment itself (such as its word contents, and that the buttons perform the *generic* behaviors that they're supposed to).

Reply:
Okay. I will use this insight to start working on it right away.

Reply:
This is still unresolved.